src/relay/transforms/partition_graph.cc

```diff
@@ -116,9 +116,11 @@ class Partitioner : public MixedModeMutator {
     for (auto f : module->functions) {
       GlobalVar f_var = f.first;
       BaseFunc f_func = f.second;
+      std::string f_name = f_var.as<GlobalVarNode>()->name_hint;
```

Comment:
Better to have a set here to make sure the function names are unqiue. Something like:
```
while (func_names.find(f_name) != func_names.end()) {
    f_name += "_a";
}
func_names.insert(f_name);
```